.kres.yaml

```diff
@@ -151,7 +151,7 @@ spec:
         - not-on-tag
       steps:
         - name: build
-          command: talosctl-all kernel sd-boot sd-stub initramfs installer-base imager talos
+          command: talosctl-multiarch-bundle talosctl-all kernel sd-boot sd-stub initramfs installer-base imager talos
```

Comment:
probably we don't need to build an image here, and just leave `talosctl-all` which will save `talosctl` binary in the artifacts

(building on `push` should be enough)